app/controllers/posts_controller.rb

```diff
@@ -1,4 +1,6 @@
 class PostsController < ApplicationController
```

Comment:
- [ ] It will be great to add `load_and_authorize_resource` at the top of the posts controller. This method is provided to automatically authorize all actions in a RESTful style. It will use a before action to load the resource into an instance variable and authorize it for every action.

Reply:
Thank you for your suggestion. I didn't see this point in the requirements, but it works great! Thanks

Reply:
- `load_and_authorize_resource` has been successfully added to the PostController 👍🏽